style.css

```diff
@@ -0,0 +1,207 @@
+body {
+  width: 375px;
+  height: 5147px;
+  background: #f7f7f9;
+}
+
+.header-bg {
+  position: relative;
+  height: 718px;
+  width: 375px;
+  left: 0;
+  right: 0;
+  border-radius: 0 0 0 75px;
+  background: #fff;
+}
+
+#appBar {
+  position: absolute;
+  width: 375px;
+  height: 92px;
+  top: 0;
+  left: 0;
+}
+
+#toolsBar {
+  position: absolute;
+  top: 44px;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  align-content: center;
+  justify-content: center;
+}
+
+.logoPlaceHolder {
+  position: absolute;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 10px 15px;
+  width: 100px;
+  height: 15px;
+  top: 6px;
+  left: 12px;
+  justify-items: center;
+}
+
+.logo {
+  width: 76px;
+  height: 20px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 20px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  color: #6070ff;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  text-decoration: none;
+}
+
+#Hamburger {
+  position: absolute;
+  display: flex;
+  flex-direction: row;
+  align-items: flex-start;
+  justify-content: center;
+  padding: 4px;
+  width: 30px;
+  height: 30px;
+  right: 24px;
+  top: 6px;
+  border-radius: 8px;
+}
+
+.bar {
+  display: block;
+  width: 25px;
+  height: 3px;
+  margin: 5px auto;
+  -webkit-transition: all 0.3s ease-in-out;
+  transition: all 0.3s ease-in-out;
+  background-color: #6070ff;
+}
+
+.headline {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 114px 24px;
+  gap: 10px;
+  position: absolute;
+  width: 375px;
+  height: 604px;
+  top: 92px;
+  background-image: url(../Portfolio/asset/images/headershape.svg);
+  background-repeat: no-repeat;
+  background-position-y: 15px;
```

Comment:
- [ ] Kindly **as suggested by the previous reviewer,** the background image is not visible on the webpage, You may try to fix this issue to replicate the Figma design 🙏 

**Your design** | **Figma design**
| --- | --- |
![image](https://user-images.githubusercontent.com/102757126/219689293-10856a92-aae7-45eb-b3f8-a4c9fcd7f6e1.png) | ![image](https://user-images.githubusercontent.com/102757126/219689401-aae5c29c-13db-4661-90e0-e988b1044432.png)

Reply:
I think the background image is from your end as i can view see on my browser here